modMcf/src/org/aion/mcf/config/CfgSync.java

```diff
@@ -43,13 +43,24 @@
     private boolean showStatus;
     private Set<StatsType> showStatistics;
 
-    private static int BLOCKS_QUEUE_MAX = 32;
+    private boolean compactEnabled;
+    private int slowImportTime;
+    private int compactFrequency;
+
+    private static final int BLOCKS_QUEUE_MAX = 32;
+
+    private static final int SLOW_IMPORT_TIME = 1_000; // 1 sec
+    private static final int COMPACT_FREQUENCY = 600_000; // 10 min
+
```

Comment:
Quirk: My code formatter takes out this extra newline when I run it. Is it just me?